scripts/cross/bootstrap-centos.sh

```diff
@@ -3,3 +3,5 @@ set -o errexit
 
 yum install -y unzip centos-release-scl
 yum install -y llvm-toolset-7
+yum install -y perl-IPC-Cmd
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏ this is an odd new dependency 🤔

Reply:
Indeed it is, it's need to compile openssl though. I can add a comment.